microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -26,30 +24,28 @@ CSS Grid is a powerful layout paradigm for the web.  A great place to get starte
 <!-- ====================================================================== -->
 ## Discover CSS grids
 
-When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a `grid` badge is displayed next to it in the **Elements** tool:
+When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a **grid** badge is displayed next to it in the **Elements** tool:
 
 ![Discover grid](./grid-images/grid-discover-grid.png)
 
-Click the badge to toggle the display of a grid overlay on the page.  The overlay appears over the element, laid out like a grid to display the position of the grid lines and tracks:
+Click the **grid** badge next to an element to display a grid overlay over that element in the rendered webpage.  The overlay appears over the element, and shows the position of the grid lines and tracks:
 
 ![Toggle grid badge](./grid-images/grid-highlight-grid.png)
 
-Open the **Layout** pane.  When grids are included on a page, the **Layout** pane includes a **Grid** section containing a number of options for viewing the grids.
+When grids are included on a page, the **Layout** pane in the **Elements** tool includes a **Grid** section that lists the grid containers and lets you configure the information which grid overlays should display.
```

Comment:
```suggestion
When grids are included on a page, the **Layout** pane in the **Elements** tool includes a **Grid** section that lists the grid containers and lets you configure which information to display in grid overlays.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which - re: style guides: the question of applicability

Reply:
result, conforming to style guide; active verb: "Use foo to bar", instead of "lets you".  @captainbrosset 
```
When grids are included on a webpage, the **Layout** pane in the 
**Elements** tool includes a **Grid** section that lists the grid containers:

![Layout pane](./grid-images/grid-layout-pane.png)

Use the **Grid** section to configure which information to display 
in grid overlays on the rendered webpage.  
The **Grid** section in the **Layout** pane contains the following subsections:
```